### PR TITLE
New bloodpack sprites

### PR DESCRIPTION
**Pull request by UbaserB**
New blood pack sprites, including unused assets for when bloodpacks can be coded to work similar to a container.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image
:cl: Ubaser
- tweak: Nanotrasen has decided to purchase their blood packs from a different company, so they now contain real blood.

### CODE REVIEW

**Comment by UbaserB** [comment]
fix'd

**Comment by UbaserB** [comment]
also dont tell me it looks like a chair or a juicebox please

**Comment by lzk228** [comment]
there's out of game style imo
and why was it blue? because it is a blood substitute that is suitable for all races

**Comment by potato1234x** [comment]
it looks way too shiny

**Comment by UbaserB** [comment]
im working on a new one right now, its on hold for a rework

**Comment by QuietlyWhisper** [comment]
> there's out of game style imo and why was it blue? because it is a blood substitute that is suitable for all races

Correct

**Comment by AJCM-git** [comment]
> im working on a new one right now, its on hold for a rework

are you still working on new ones? (if so please make this a draft)

**Comment by UbaserB** [comment]
uhh its half done i havent made much progress in a while though. Ill make it a draft for now.

File: Resources/Textures/Objects/Specific/Medical/medical.rsi/meta.json
Comment (ficcialfaint) [inline Resources/Textures/Objects/Specific/Medical/medical.rsi/meta.json:1]: Please fix the indents, commas

**Review by ficcialfaint — DISMISSED** [review]

File: Resources/Textures/Objects/Specific/Medical/medical.rsi/meta.json
Comment (UbaserB) [reply Resources/Textures/Objects/Specific/Medical/medical.rsi/meta.json:1]: aaaaaa i will

**Comment by QuietlyWhisper** [comment]
the purpose of the blood packs being blue is they're available to be used for any species. there is a red blood pack sprite already, though this new one (at the time of writing) is a lot nicer!

**Comment by UbaserB** [comment]
> the purpose of the blood packs being blue is they're available to be used for any species. there is a red blood pack sprite already, though this new one (at the time of writing) is a lot nicer!

ill put the blue variant unused in files then :)

**Comment by UbaserB** [comment]
oh right i forgot to add the metadata i can do that later today. I believe i have already explained this in my PR, but i added it for when it inevitably will be added because people like emisse wanted me to implement this function but i toned it down to simply a sprite rework since i cant code.

**Comment by lzk228** [comment]
> I believe i have already explained this in my PR

oh, didn't read it :trollface:

**Comment by deltanedas** [comment]
evil

**Pull request merged**